test/functional/page_objects/settings_page.js

```diff
@@ -22,50 +22,40 @@ import { map as mapAsync } from 'bluebird';
 export function SettingsPageProvider({ getService, getPageObjects }) {
   const log = getService('log');
   const retry = getService('retry');
-  const config = getService('config');
   const remote = getService('remote');
   const find = getService('find');
   const flyout = getService('flyout');
   const testSubjects = getService('testSubjects');
   const PageObjects = getPageObjects(['header', 'common', 'visualize']);
 
-  const defaultFindTimeout = config.get('timeouts.find');
-
   class SettingsPage {
     async clickNavigation() {
-      // TODO: find better way to target the element
-      await remote.findDisplayedByCssSelector('.app-link:nth-child(5) a').click();
-    }
-
-    async clickLinkText(text) {
-      await retry.try(async () => {
-        await remote.findDisplayedByLinkText(text).click();
-      });
+      find.clickByCssSelector('.app-link:nth-child(5) a');
```

Comment:
Should be `find.clickDisplayedByCssSelector('...');` but that function doesn't actually exist, so I think we need a new one.  We do have `find.existsByDisplayedByCssSelector` but not an equivalent click one.